subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/TestFilter.java

```diff
@@ -51,6 +51,12 @@
  *
  *          //specific test class and test method
  *          includeTest "org.gradle.SomeTest", "someTestMethod"
+ *
+ *          //special case without any wildcards, this can match 'SomeTest' class under any package
```

Comment:
⭕️ We don't need to show every possible combination. Just show the ones that actually have an important use case like "I want to run just this test".